src/shared/src/types/team-types.ts

```diff
@@ -16,4 +16,4 @@ export interface Team {
   projects: ProjectPreview[];
 }
 
-export type TeamPreview = Pick<Team, 'teamId' | 'teamName'>;
+export type TeamPreview = Pick<Team, 'teamId' | 'teamName' | 'members'>;
```

Comment:
if team previews are now sending members, you don't need to filter by that anymore you can just do team.members.map() instead of doing all users

Reply:
facts